synapseAnnotations/data/analysis.json

```diff
@@ -568,106 +515,88 @@
-        "value": "penalizedLogisticRegression", 
-        "description": "", 
-        "source": ""
-      }, 
-      {
-        "value": "penalizedLinearRegression", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "penalized regression",
+        "description": "A regression model with a parametric constraint on the allowable solution space.",
+        "source": "https://statweb.stanford.edu/~tibs/lasso/lasso.pdf"
+      },
       {
-        "value": "randomForest", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "random forest",
+        "description": "A method to build an ensemble of decisions trees for classification or regression models.",
+        "source": "http://ect.bell-labs.com/who/tkh/publications/papers/odt.pdf"
+      },
       {
-        "value": "supportVectorMachine", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "support vector machine",
+        "description": "A support vector machine is a data transformation with a class prediction objective based on the construction of a separating hyperplane that maximizes the margin between two data sets of vectors in n-dimensional space.",
+        "source": "http://purl.obolibrary.org/obo/OBI_0000700"
+      },
       {
-        "value": "deepLearning", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "deep learning",
+        "description": "Rebranded artificial neural networks.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/25462637"
+      },
       {
-        "value": "boosting", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "boosting",
+        "description": "An ensemble meta algorithm to average over many predictions to reduce bias and variance in predictions.",
+        "source": "https://web.archive.org/web/20150119081741/http://oz.berkeley.edu/~breiman/arcall96.pdf"
+      },
       {
-        "value": "bagging", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "bagging",
+        "description": "An ensemble method to stabilize predictions, improve accuracy, and reduce variance for regression or decision tree models.",
+        "source": "https://link.springer.com/article/10.1007%2FBF00058655"
+      },
       {
-        "value": "naiveBayes", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "naive Bayes",
+        "description": "A classification model that assumes independence of predictors and uses Bayes theorem predict outcomes.",
+        "source": "http://engr.case.edu/ray_soumya/mlrg/optimality_of_nb.pdf"
+      },
       {
-        "value": "GaussianProcessRegression", 
-        "description": "", 
-        "source": ""
+        "value": "Gaussian process regression",
+        "description": "A non parametric kernel based supervised learning regression method.",
+        "source": "http://www.inference.org.uk/itprnn/book.pdf"
```

Comment:
Ah, the classic "just cite the whole textbook" technique

Reply:
So we can just cite Judea Pearl for anything Bayesian, while we're at it, @blogsdon ?

Reply:
Wait, in the contributing docs does it explicitly say you can't cite a text book?

Reply:
It does not! I would, however, make this a proper citation. I don't have a specific formatting preferences, but something that at least gets the  author, year, edition, page number in there. I would also link to http://www.inference.org.uk/mackay/itila/ instead.